turbinia/client.py

```diff
@@ -131,6 +131,23 @@ def get_turbinia_client(run_local=False):
     raise TurbiniaException(msg)
 
 
+def register_job_timeouts(dependencies):
+  """Registers a timeout for each job.
+
+  Args:
+    dependencies(dict): dependencies to grab timeout value from.
+  """
+  log.info('Registering job timeouts.')
+
+  job_names = list(job_manager.JobsManager.GetJobNames())
+  # Iterate through list of jobs
+  for job, values in dependencies.items():
+    if job not in job_names:
+      continue
+    timeout = values.get('timeout')
```

Comment:
Should we add a default here (and maybe a log line if we use the default)?  Otherwise if any of the jobs don't have a `timeout` value in the config, they will get `None` and might crash the worker when they try to use that.